### PR TITLE
Maintain client v1 and v2 versions

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita sharma <harshita.sharma6174@gmail.com>

**What this PR does / why we need it**: Creating separate file paths for v1 and v2 client versions.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/agent/kubermatic/v1/agent.go [hunk 1]
```diff
@@ -225,29 +227,16 @@ func clusterFromKube(kn kubermaticv1.Cluster, seedName string) (Cluster, error)
...
 		opaEnabled = opaIntegration.Enabled
 	}
 
-	var userSSHKeyAgentEnabled bool
+	var enableUserSSHKeyAgent bool
 	enableUserSSHKeyAgentPointer := kn.Spec.EnableUserSSHKeyAgent
-	if enableUserSSHKeyAgentPointer != nil {
-		userSSHKeyAgentEnabled = *enableUserSSHKeyAgentPointer
+	if kn.Spec.EnableUserSSHKeyAgent != nil {
+		enableUserSSHKeyAgent = *enableUserSSHKeyAgentPointer
```
Comment (lsviben) [inline pkg/agent/kubermatic/v1/agent.go:236]:
```suggestion
	if kn.Spec.EnableUserSSHKeyAgent != nil {
		enableUserSSHKeyAgent = *kn.Spec.EnableUserSSHKeyAgent
```

File: pkg/report/v2/report.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 The Telemetry Authors.
```
Comment (lsviben) [inline pkg/report/v2/report.go:2]: year should be 2023, please check new boilerplate for this :)

**Review by lsviben — COMMENTED** [review]
minor comments

File: pkg/agent/kubermatic/v1/agent.go (hunk 1 above)
Comment (imharshita) [reply pkg/agent/kubermatic/v1/agent.go:243]: but why not use variable, it is the same thing right?

Comment (lsviben) [reply pkg/agent/kubermatic/v1/agent.go]: i mean its just shorter, but personal preference i guess

Comment (imharshita) [reply pkg/agent/kubermatic/v1/agent.go]: Yes. So should i change this or not?

Comment (lsviben) [reply pkg/agent/kubermatic/v1/agent.go]: if you prefer it as it was, leave it. I just gave my suggestion :)

Comment (imharshita) [reply pkg/agent/kubermatic/v1/agent.go]: Okay please approve the pr if it looks good

**Pull request merged**